telegram/ext/_application.py

```diff
@@ -1061,7 +1072,12 @@ async def _update_fetcher(self) -> None:
 
             if self._concurrent_updates:
                 # We don't await the below because it has to be run concurrently
-                self.create_task(self.__process_update_wrapper(update), update=update)
```

Comment:
`__process_update_wrapper` had also marked the task as done in the update queue (`self.update_queue.task_done()`), shouldn't we do that too in `do_process_update`?

Reply:
done